projects/saturn/src/main/java/io/fairspace/saturn/services/views/SparqlQueryService.java

```diff
@@ -364,7 +364,9 @@ private static NodeValue toNodeValue(Object o, ColumnType type) {
             case Identifier, Term, TermSet -> makeNode(createURI(o.toString()));
             case Text, Set -> makeString(o.toString());
             case Number -> makeDecimal(o.toString());
-            case Date -> makeDateTime(convertDateValue(o.toString()));
+                // Extract date only as it comes in format "yyyy-MM-dd'T'HH:mm:ss.SSSX"
+                // Leave it as is in case of adding new DateTime filter format
+            case Date -> makeDate(o.toString().split("T")[0]);
```

Comment:
I haven't tested it, but wouldn't it leave `yyyy-MM-dd'` after the split (with an apostrophe `'` at the end)? 

But it looks like the format that it actually comes in is without `'` `'` around `T`, right?

Reply:
It's confusing, agree. I removed the quotes in the comment as request does not contain them. It was an unlucky attempt to highlight